install.sh

```diff
@@ -23,9 +23,12 @@ else
     git clone https://github.com/PiSupply/JustBoom.git
     mkdir /opt/justboom
     cp $JustBoomDir/jb-rotary.py /opt/justboom
-    cp $JustBoomDir/jb-rotary.service /etc/systemd/system
+    cp $JustBoomDir/jb-rotary.service /usr/lib/systemd/system
+    cp $JustBoomDir/jb-rotary.timer /usr/lib/systemd/system
```

Comment:
Are these paths correct? Should they not be /lib/systemd/system not /usr/lib/systemd/system